pkg/ctl/utils/associate_iam_oidc_provider.go

```diff
@@ -22,6 +22,9 @@ func associateIAMOIDCProviderCmd(cmd *cmdutils.Cmd) {
 	}
 
 	cmd.FlagSetGroup.InFlagSet("General", func(fs *pflag.FlagSet) {
+
+		fs.StringSliceVar(&cfg.IAM.OIDCClientIDList, "oidc-client-id-list", []string{}, "List of Client ID for OIDC provider (audience)")
```

Comment:
perhaps rename `oidc-client-id-list` to `oidc-client-ids`

Reply:
Please check how other util commands add flags. Normally, the value of the flag is not stored directly into the `cfg`(`ClusterConfig`) as I think it can be overwritten later by the loader. It should be stored in a variable first and then passed to the command or to the loader.